src/chunk_list.cpp

```diff
@@ -9,34 +9,171 @@
 #include <cstring>
 #include <cstdlib>
 
+#include "uncrustify_types.h"
 #include "ListManager.h"
 #include "prototypes.h"
 #include "uncrustify.h"
 #include "space.h"
 
 
-typedef ListManager<chunk_t> ChunkList;
+typedef ListManager<chunk_t> ChunkList_t;
+
+/** use this enum to define in what direction or location an
+ * operation shall be performed. */
+enum loc_t
+{
+   before, /**< indicates a position or direction upwards (=pref) */
```

Comment:
Yes, the enums should be improved, but I don't like adding pointless prefixes so I will make them all caps like the majority of the enums in the codebase.